chore(README): update to include bloom filter (#9)
LibertyDSNP/parquetjs: README.md

Once we are finished adding rows to the file, we have to tell the writer object
to flush the metadata to disk and close the file by calling the `close()` method:

### Adding bloom filters 

Bloom filters can be added to multiple columns as demonstrated below:

``` js
  const options = {
    bloomFilters: [
      {
        column: "name",
        numFilterBytes: 1024,
      },
      {
        column: "quantity",
        numFilterBytes: 1024,
      },
    ]
  };

var writer = await parquet.ParquetWriter.openFile(schema, 'fruits.parquet', options);
```

> By default, not passing any additonal options calculates the optimal number of blocks according to the number of distinct values and false positive probability.

Reviewer: ```suggestion
By default, not passing any additonal options calculates the optimal number of blocks according to the default number of distinct values of 128*1024 and default false positive probability of 0.001.
```

await reader.close();
```

### Reading bloom filter

Bloom filters can be fetched from a parquet file by creating a reader
and calling `getBloomFiltersFor`.
``` js
// create new ParquetReader that reads from 'fruits.parquet`
let reader = await parquet.ParquetReader.openFile('fruits.parquet');

// fetches bloom filter for the columns provided.
const bloomFilters = reader.getBloomFiltersFor(['name']);

=> {
  name: [
    {
      rowGroupIndex: 0
      columnName: 'name',
      sbbf: SplitBlockBloomFilter<instance>
    }
  ]
}

```

> Calling `getBloomFiltersFor` on reader returns an object with the keys being a column name and value being an array which length is the number of row groups that the column spans.

Reviewer: ```suggestion
Calling `getBloomFiltersFor` on the reader returns an object with the keys being a column name and value being an array of length equal to the number of row groups that the column spans.
```
